server/crawler/index.ts

```diff
@@ -196,9 +204,19 @@ export default async function getCralwerData(url: string) {
   let body = ''
   let ogImage = ''
   let images:any = []
+  let browser:any
+  let page:any
+  let content:any
 
   try {
-    const { data: content } = await axios.get(encodeURI(url as string))
+    await axios.get(encodeURI(url as string),{ withCredentials: true }).then((element)=>{
+      content = element.data
+    }).catch(async () => {
+        ({ browser } = await getBrowserPage());
```

Comment:
Please don't catch all error, only trigger `puppeteer` if the error is `403` and and it is a cloudflare page.